views/backend/postback.php

```diff
@@ -1,109 +1,21 @@
-<?php
-    if($model_user->admin()!=1){
-        header("Location:/dashboard");
-    }
-
-    $post = $model_article->get_post();
-    if($post == false){
-        header("Location:/page-erreur-administrateur");
-    }
-?>
-
 <h2>Modifier un article</h2>
         <div class="row center">
             <div class="row">
-                <img class="responsive-img" src="/public/img/posts/<?= $post->image ?>" alt="<?= $post->title ?>"/>
+                <img class="responsive-img" src="/public/img/posts/<?= $post->getImage() ?>" alt="<?= $post->getTitle() ?>"/>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)